src/main/scala/org/ergoplatform/nodeView/history/storage/modifierprocessors/ToDownloadProcessor.scala

```diff
@@ -54,10 +56,13 @@ trait ToDownloadProcessor extends BasicReaders with ScorexLogging {
     }
 
     bestFullBlockOpt match {
+        //do not download full blocks if no headers-chain synced yet or SPV mode
```

Comment:
let's move comments inside the body of the corresponding branch.

Reply:
done